src/core/hle/service/time/system_clock_core.h

```diff
@@ -0,0 +1,66 @@
+// Copyright 2019 yuzu emulator team
+// Licensed under GPLv2 or any later version
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include "common/common_types.h"
+#include "core/hle/service/time/clock_types.h"
+
+namespace Core {
+class System;
+}
+
+namespace Service::Time::Clock {
+
+class SteadyClockCore;
+class SystemClockContextUpdateCallback;
+
+class SystemClockCore {
+public:
+    SystemClockCore(SteadyClockCore& steady_clock_core);
```

Comment:
```suggestion
    explicit SystemClockCore(SteadyClockCore& steady_clock_core);
```

The destructor should likely be defaulted in the cpp file as well, since the class contains a non-trivial member (the shared_ptr instance).